nuxt/api/draftLyrics.ts

```diff
@@ -0,0 +1,56 @@
+import type { NuxtAxiosInstance } from '@nuxtjs/axios';
+import { Format } from '@/entities/lyrics';
+import { DraftLyrics } from '@/entities/draftLyrics';
+
+export interface StoreDraftLyricsPayload {
+  track_id: string;
+  document: {
```

Comment:
Might make sense to the type definition for `Document` into its own type?

```ts
type Document = {
  content: string;
  format: Format;
}
```
and reference it in the Store and Update payload interfaces?

Reply:
Looks like this is already defined in `entities/lyrics` actually. Maybe we do:

```ts
// @/entities/lyrics.ts
export interface LyricsDocument = {
  content: string;
  format: Format; 
}

export interface Lyrics extends LyricsDocument {
  /** @deprecated */
  id: string;
  /** @deprecated */
  trackId: string;
  /** @deprecated */
  createdAt: string;
  /** @deprecated */
  updatedAt: string;
}

Reply:
The problem with the existing type is that it requests for id, trackId, createAt and updatedAt hence why I created a new one, however happy to create a new interface for `Document`

Reply:
Changed the store/update methods to use LyricsDocument and created new interface LyricsDocument